packages/tools/devtools/devtools-view/src/components/TelemetryView.tsx

```diff
@@ -262,11 +313,15 @@ function FilteredTelemetryView(props: FilteredTelemetryViewProps): React.ReactEl
 		]);
 		// Initially matching options are all options
 		setMatchingOptions(eventNameOptionsRef.current);
-
 		const filtered = getFilteredEvents();
 		setFilteredTelemetryEvents(filtered);
+		if (filtered !== undefined) {
```

Comment:
`if (filtered)`

Reply:
(our linter rules _should_ enforce this)